cmd/install/install.go

```diff
@@ -74,6 +74,16 @@ func installers() (i []installer) {
 	return
 }
 
+func zshrcPath() string {
+	if zDotDir := os.Getenv("ZDOTDIR"); zDotDir != "" {
+		f := filepath.Join(zDotDir, ".zshrc")
+		if info, err := os.Stat(f); err != nil || !info.IsDir() {
```

Comment:
`err == nil && !info.IsDir()`?

Reply:
Yes, this is completely wrong. Updated it